sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py

```diff
@@ -1252,7 +1253,9 @@ def message_content():
                 message_1st_received_cnt = 0
                 message_2nd_received_cnt = 0
                 while message_1st_received_cnt < 20 or message_2nd_received_cnt < 20:
-                    messages = await receiver.receive_messages(max_batch_size=20, max_wait_time=5)
+                    messages = []
+                    async for message in receiver.get_streaming_message_iter(max_wait_time=5):
+                        messages.append(message)
```

Comment:
Actually the intention of this test case is to test the method `receive_messages` to see batch receiving is working as expected.
I think it should not be causing errors in CI after I tweaked it last time, or it's still failing consistently?

Reply:
Not consistent at all; just flaky in the sense of the typical <received one and not the rest>.  This is actually what prompted me to raise the things about min_batch_size and the max_wait_time semantics to you the other day.  

If we wanted to test receive_messages as a batch; can adjust to do that, just call it in a loop; would that balance the semantics you want to test and the robustness I'm hoping to achieve?